source/operand.h

```diff
@@ -27,7 +27,7 @@
 //
 // As we parse an instruction in text or binary form from left to right,
 // we pull and push from the front of the pattern.
```

Comment:
"we pop and push at the end of the pattern vector.   Symbols later in the pattern vector are matched against the input before symbols earlier in the pattern vector are matched."

Probably also worth saying why:
"Using a vector in this way reduces memory traffic, which is good for performance."